guides/wide-ep-lws/manifests/modelserver/gke/kustomization.yaml

```diff
@@ -1,6 +1,329 @@
+            #################
+            ${VLLM_HOME}/bin/vllm serve \
+            deepseek-ai/DeepSeek-R1 \
+            --port 8000 \
+            --enable-log-requests \
+            --enable-expert-parallel \
+            --data-parallel-hybrid-lb \
+            --tensor-parallel-size 1 \
+            --data-parallel-size $((LWS_GROUP_SIZE * DP_SIZE_LOCAL)) \
+            --data-parallel-size-local $DP_SIZE_LOCAL \
+            --data-parallel-address ${LWS_LEADER_ADDRESS} \
+            --data-parallel-rpc-port 5555 \
+            --data-parallel-start-rank ${START_RANK} \
+            --trust-remote-code \
+            --kv_transfer_config '{"kv_connector":"NixlConnector","kv_role":"kv_both"}'
+  # decode launch
+  - target:
+      kind: LeaderWorkerSet
+      name: wide-ep-llm-d-decode
+    patch: |-
+      # Replace command and args
+      - op: replace
+        path: /spec/leaderWorkerTemplate/workerTemplate/spec/containers/0/args
+        value:
+          - |
+            if command -v apt-get >/dev/null 2>&1; then
+              export VLLM_HOME=/app/venv
+            else
+              # Assume we're using the RH midstream image
+              export VLLM_HOME=/opt/vllm
+              export HF_HUB_OFFLINE=0
+            fi
+
+            # Configure gIB
+            export PATH=/usr/local/nvidia/bin:${PATH}:/usr/local/gib/bin
+            source /usr/local/gib/scripts/set_nccl_env.sh
+            unset NCCL_NET
+            export NCCL_NET_PLUGIN=/usr/local/gib/lib64/libnccl-net_internal.so
+            export START_RANK=$(( ${LWS_WORKER_INDEX:-0} * DP_SIZE_LOCAL ))
+
+            #################
+            # Decode launch
+            #################
+            ${VLLM_HOME}/bin/vllm serve \
+            deepseek-ai/DeepSeek-R1 \
+            --port 8200 \
+            --enable-log-requests \
+            --enable-expert-parallel \
+            --data-parallel-hybrid-lb \
+            --tensor-parallel-size 1 \
+            --data-parallel-size $((LWS_GROUP_SIZE * DP_SIZE_LOCAL)) \
+            --data-parallel-size-local $DP_SIZE_LOCAL \
+            --data-parallel-address ${LWS_LEADER_ADDRESS} \
+            --data-parallel-rpc-port 5555 \
+            --data-parallel-start-rank ${START_RANK} \
+            --trust-remote-code \
+            --kv_transfer_config '{"kv_connector":"NixlConnector","kv_role":"kv_both"}'
+  # Common configurations for prefill and decode.
+  - target:
+      kind: LeaderWorkerSet
+    patch: |-
+      # Add GKE specific node affinity to match H200 GPUs.
+      - op: add
+        path: /spec/leaderWorkerTemplate/workerTemplate/spec/affinity
+        value:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: cloud.google.com/gke-accelerator
+                      operator: In
+                      values:
+                        - nvidia-h200-141gb
```

Comment:
I think this is too specific for our guide, the user is expected to select working hardware and our docs should state that.  `nvidia.com/gpu: 8` is sufficient.